frontend/src/components/stylists/headerBarCard.jsx

```diff
@@ -71,33 +81,33 @@ function HeaderCardAppointmentTime(appTime) {
       <div>
         <div className="header-card-rightmost-section">
           <FontAwesomeIcon icon={faClock} />
-          <p className="app-time">{displayTime(appTime.appTime)}</p>
+          <p className="app-time">{displayTime(time)}</p>
         </div>
       </div>
     </Fragment>
   );
 }
 
 function HeaderCardDropRight(props) {
-  // state = { dropdownOptions: [] };
   const [dropdownOptions, setDropdownOptions] = useState([]);
+  const [showDropdown, setShowDropdown] = useState(false);
 
   useEffect(() => {
-    getStylistsForDropdown();
+    let activeUser = JSON.parse(sessionStorage.getItem("user"));
+    if (activeUser.role === 0 || activeUser.role === 3) {
+      setShowDropdown(true);
+      getStylistsForDropdown();
+    }
   }, []);
 
   const getStylistsForDropdown = async () => {
-    // TODO: ADD ROUTE TO GET USERS THAT ARE STYLISTS ONLY.
-    console.log("getting stylists from backend...");
     try {
-      console.log(props);
       let response = await axios.get(props.backendDomain + "stylist", {
         headers: {
-          Authorization: `basic ${sessionStorage.getItem("authToken")}`,
+          Authorization: "JWT " + localStorage.getItem("token"),
```

Comment:
You could write this line like you had it before (enclosed in these ``):

`JWT ${localStorage.getItem("token")}`

Just a suggestion, not necessary since it would do the exact same thing.